.clang-format

```diff
@@ -36,7 +36,7 @@ BreakBeforeBinaryOperators: None
 BreakBeforeBraces: Linux
 BreakBeforeTernaryOperators: true
 BreakConstructorInitializersBeforeComma: false
-ColumnLimit:     85
+ColumnLimit:     100
```

Comment:
We could give 90 a try, this should still work on GitHub and it works on my desktop screens. For the laptop, I don't use that so often for QPhiX anyway, so that could be a compromise?